unity/ggj20/Assets/Scripts/AI/Wander.cs

```diff
@@ -0,0 +1,84 @@
+﻿using UnityEngine;
+using System.Collections;
+ 
+[RequireComponent(typeof(CharacterController))]
+public class Wander : MonoBehaviour
+{
+	public float speed = 5;
+	public float directionChangeInterval = 1;
+	public float maxHeadingChange = 30;
+ 
+	CharacterController controller;
+	float heading;
+	Vector3 targetRotation;
+ 
+	void Awake ()
+	{
+		controller = GetComponent<CharacterController>(); 
+		heading = Random.Range(0, 360);
+		transform.eulerAngles = new Vector3(0, heading, 0);
+ 
+		StartCoroutine(NewHeading());
+	}
+ 
+	void Update ()

```

Comment:
I wonder if this calculation, since it involves transforms, should be in `FixedUpdate()` instead.